docs/how-tos/control-motor.md

```diff
@@ -51,6 +51,8 @@ If you do not have both a board and motor, install `viam-server` on your laptop
 
 ## Instructions
 
+Make sure to wire your motor to your board before starting.
+Power the board on if you want to test your machine while configuring it.
```

Comment:
I think this makes just as much sense as putting it in requirements, but I'd suggest picking one or the other for all of these for consistency. Or is there a reason this one is here instead?